StarCraft2.ahk

```diff
@@ -131,11 +138,14 @@ class SC2
         this.tankyPos := yPos
         this.casterxPos := xPos + 188    ; compared to the center: a little bit to the right
         this.casteryPos := yPos - 120    ; compared to the center: a little bit upper
+        this.cryoxPos := xPos - 20    ; compared to the center: a little bit to the right
```

Comment:
compared to the center: a little bit to the `left`

Reply:
That's so the Cyro throws the Cyro missiles slightly behind him, keeping the zombies close enough to get hit by most of everything.